src/number_theory/padics/padic_integers.lean

```diff
@@ -80,26 +81,25 @@ def subring : subring (ℚ_[p]) :=
 
 variables {p}
 
-/-- Addition on ℤ_p is inherited from ℚ_p. -/
+/-- Addition on `ℤ_[p]` is inherited from `ℚ_[p]`. -/
 instance : has_add ℤ_[p] := (by apply_instance : has_add (subring p))
 
-/-- Multiplication on ℤ_p is inherited from ℚ_p. -/
+/-- Multiplication on `ℤ_[p]` is inherited from `ℚ_[p]`. -/
 instance : has_mul ℤ_[p] := (by apply_instance : has_mul (subring p))
 
-/-- Negation on ℤ_p is inherited from ℚ_p. -/
+/-- Negation on `ℤ_[p]` is inherited from `ℚ_[p]`. -/
 instance : has_neg ℤ_[p] := (by apply_instance : has_neg (subring p))
 
-/-- Subtraction on ℤ_p is inherited from ℚ_p. -/
+/-- Subtraction on `ℤ_[p]` is inherited from `ℚ_[p]`. -/
 instance : has_sub ℤ_[p] := (by apply_instance : has_sub (subring p))
 
-/-- Zero on ℤ_p is inherited from ℚ_p. -/
+/-- Zero on `ℤ_[p]` is inherited from `ℚ_[p]`. -/
 instance : has_zero ℤ_[p] := (by apply_instance : has_zero (subring p))
 
 instance : inhabited ℤ_[p] := ⟨0⟩
 
-/-- One on ℤ_p is inherited from ℚ_p. -/
-instance : has_one ℤ_[p] :=
-⟨⟨1, by norm_num⟩⟩
+/-- One on `ℤ_[p]` is inherited from `ℚ_[p]`. -/
```

Comment:
```suggestion
/-- `has_one` on `ℤ_[p]` is inherited from `ℚ_[p]`. -/
```

Reply:
Just saying "One" is consistent with the other instances above it. If we were to change this, then we should also change "Addition" to "`has_add`", etc.

Reply:
OK, let's keep it.